Nitrox.Assets.Subnautica/LanguageFiles/en.json

```
@@ -65,5 +65,7 @@
     "Nitrox_WaitingPassword": "Waiting for server password input…",
     "Nitrox_WaitingUserInput": "Waiting for user input…",
     "Nitrox_WorldSettling": "Awaiting World Settling…",
-    "Nitrox_RemotePlayerObstacle": "Another player's either inside or too close to the deconstructable target."
+    "Nitrox_RemotePlayerObstacle": "Another player's either inside or too close to the deconstructable target.",
+    "Nitrox_PlayerJoined": "{PLAYER} joined the game.",
+    "Nitrox_PlayerLeft": "{PLAYER} joined the game."
```

Typo in Nitrox_PlayerLeft:
```suggestion
    "Nitrox_PlayerLeft": "{PLAYER} left the game."
```

```